index.html

```diff
@@ -257,7 +257,7 @@ <h2 id="l1" class="about-me-languages">Language</h2>
             </ul>
         </div>
         <div class="desktop-skills-content">
-            <img id="rect2" src="images/Rectangle 56.png">
+            <img id="rect2" src="images/Rectangle 56.png" alt="shape 2">
```

Comment:
- [ ] Kindly consider using a descriptive `alternative text` for your images, That tells what the image is about.

Reply:
Thank you for pointing this out. I have changed the alt text for these shapes.